blog/2024-02-01-introducing-dewy/index.mdx

```diff
@@ -12,8 +12,8 @@ import TabItem from '@theme/TabItem';
 
 
 
-Today we're releasing the first version of [Dewy](https://dewykb.github.io/dewy), a knowledge base built for the specific needs of Gen AI applications.
```

Comment:
Updated a bunch of links -- they were using absolute instead of relative, so they didn't work on the locally hosted app. Also, it caused things to bypass the dead-link detection that would have caught some problems.